echidna/core/dsextract.py

```diff
@@ -73,19 +78,25 @@ class EnergyExtractMC(Extractor):
       fv_radius (float, optional): Fiducial radius. Applies a cut to remove
         events which have a radial position greater than the radius of the
         fiducial volume. If None no cut is applied.
-      reco_pos (bool, optional): If true then position cuts will be made on
+      external (bool, optional): Set to True for external backgrounds.
+        Default is False.
+      reco_pos (bool, optional): If True then position cuts will be made on
         reconstructed position. If False (default) then MC position is used
         for cuts.
+      external (bool, optional): Set to True for external backgrounds.
+        Default is False.
 
     Attributes:
-      _reco_pos (bool): If true then position cuts will be made on
+      _reco_pos (bool): If True then position cuts will be made on
       reconstructed position. If False then MC position is used for cuts.
     '''
 
-    def __init__(self, fv_radius=None, reco_pos=False):
+    def __init__(self, fv_radius=None, external=False, reco_pos=False):
         '''Initialise the class
         '''
-        super(EnergyExtractMC, self).__init__("energy_mc", fv_radius)
+        super(EnergyExtractMC, self).__init__("energy_mc", fv_radius, external)
+        if external:
+            reco_pos = True  # Overwrite Default. No mc pos for external.
```

Comment:
@ashleyrback Its because mcpos for externals is the original isotope position so they will be all outside the AV so all the data will be outside the cut or fitting region.